charts/generic/k8-pod-delete/Cluster/engine-app-health.yaml

```diff
@@ -9,8 +9,7 @@ spec:
     applabel: 'app=nginx'
     appkind: 'deployment'
   engineState: 'active'
-  chaosServiceAccount: chaos-admin 
-  jobCleanUpPolicy: 'retain'
+  chaosServiceAccount: chaos-admin
```

Comment:
Do you also want to update in Azure engine and getstarted engine for pod-delete and container-kill @ispeakc0de?

Reply:
I intentionally didn't remove the jobCleanupPolicy from those places
- It should be present in get-started as the user will aware of the field
- For azure we can remove but right now we are not rendering it on the chart so I leave it as it is

Reply:
Okay, then we should also keep  `annotationCheck` in getstarted?